src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>${groupId}</groupId>
+  <artifactId>${artifactId}</artifactId>
+  <packaging>war</packaging>
+  <version>${version}</version>
+
+  <properties>
+    <webappdir>${project.build.directory}/webapp</webappdir>
+    <webapp.libdir>${webappdir}/WEB-INF/lib</webapp.libdir>
+
+    <j2cl.maven.plugin.version>0.12-SNAPSHOT</j2cl.maven.plugin.version>
+    <j2cl-compilation.level>BUNDLE</j2cl-compilation.level>
```

Comment:
we should not pick a level by default - specifying it for the whole plugin means that will be used for prod, tests, dev. instead, at best it should be specialized per-module, per-execution, at least in general